server/client_rest.go

```diff
@@ -31,6 +31,19 @@ type clientRest struct {
 	httpClient http.Client
 }
 
+func (client *clientRest) RegisterIdentity(identity *dto_discovery.Identity) (err error) {
+	response, err := client.doRequest("POST", "identities", dto.CreateIdentityRequest{
```

Comment:
I miss test, which freezes request JSON structure. I know we did not have tests for this, but we should start having